pages/api/payment-received.ts

```diff
@@ -62,30 +62,36 @@ const paymentReceivedHandler: NextApiHandler = async (req, res) => {
       where: { id: applicationId },
       select: { email: true, firstName: true },
     });
-    const applicationEmail = application?.email;
-    const applicationFirstName = application?.firstName;
 
-    // Update application
-    await prisma.application.update({
-      where: { id: applicationId },
-      data: {
-        shopifyPaymentStatus: ShopifyPaymentStatus.RECEIVED,
-        shopifyConfirmationNumber: `${shopifyOrderId}`,
-        paidThroughShopify: true,
-        donationAmount: donationAmount,
-      },
-    });
+    if (!application) {
+      // Bad request, occurs if application ID does not exist
+      res.status(400).end();
```

Comment:
```suggestion
      return res.status(400).end();
```
and then the rest of the code doesn't need to live in an `else` block

Reply:
Nice!!